qemu/tests/cfg/virtio_mode.cfg

```diff
@@ -29,14 +29,24 @@
             pci_id_pattern = "(\d+:\d+\.\d+)\s+Ethernet controller:.*?Virtio network device"
         - with_viorng:
             driver_name = viorng
-            no_virtio_rng:
-                virtio_rngs += " rng0"
-                backend_rng0 = rng-random
-                backend_type = passthrough
-                filename_passthrough = /dev/urandom
             device_type = "virtio-rng-pci"
             device_name = "VirtIO RNG Device"
             pci_id_pattern = "(\d+:\d+\.\d+)\s+Unclassified device\s.*?Virtio RNG"
+            no_virtio_rng:
+                virtio_rngs += " rng0"
+            variants:
+                - rng_builtin:
```

Comment:
There is a potential name collision problem, because its name is the same as the one in `guest_hw.cfg`, have to pick another one.

Reply:
Updated, thanks!